docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7-slim
+FROM python:3.8-slim
```

Comment:
I'm not sure we should target 3.8 considering that our userbase would most likely run 3.6 (centos 7/8) or 3.7 (debian stable). Perhaps start with 3.6?